server/config/config.go

```diff
@@ -100,6 +100,7 @@ const (
 
 	BlockExecutorSequential = "sequential"
 	BlockExecutorBlockSTM   = "block-stm"
+	DefaultMaxTxs           = 1000
```

Comment:
```suggestion
	DefaultMaxTxs           = 3000
```

Maybe increase a bit the default value, we do have some capacity in the mainnet 🤔

Reply:
Yea, but still need overwrite for testground

Reply:
No problem